LambdaEngine/Include/Rendering/RenderGraph.h

```diff
@@ -386,7 +387,7 @@ namespace LambdaEngine
 		bool CreateDescriptorHeap();
 		bool CreateCopyCommandLists();
 		bool CreateResources(const TArray<RenderGraphResourceDesc>& resourceDescriptions);
-		bool CreateRenderStages(const TArray<RenderStageDesc>& renderStages, const THashTable<String, RenderGraphShaderConstants>& shaderConstants, TSet<uint32>& requiredDrawArgs);
+		bool CreateRenderStages(const TArray<RenderStageDesc>& renderStages, const THashTable<String, RenderGraphShaderConstants>& shaderConstants, const TArray<ICustomRenderer*> customRenderers, TSet<uint32>& requiredDrawArgs);
```

Comment:
`customRenderers` should be a reference. The linter should've picked up on this.